src/main/webapp/resources/js/pages/projects/samples-metadata-import/index.js

```diff
@@ -3,11 +3,11 @@ import { BrowserRouter, Route, Routes } from "react-router-dom";
 import { Provider } from "react-redux";
 import { render } from "react-dom";
 import { SampleMetadataImportComplete } from "./components/SampleMetadataImportComplete";
-import { SampleMetadataImportMapHeaders } from "./components/SampleMetadataImportMapHeaders";
+import { SampleMetadataImportMapColumns } from "./components/SampleMetadataImportMapColumns";
 import { SampleMetadataImportReview } from "./components/SampleMetadataImportReview";
-import { SampleMetadataImportUploadFile } from "./components/SampleMetadataImportUploadFile";
+import { SampleMetadataImportSelectFile } from "./components/SampleMetadataImportSelectFile";
 import { setBaseUrl } from "../../../utilities/url-utilities";
```

Comment:
This file should be updated to TypeScript.

Reply:
Please see 0eb00c18859b9254fe32d98744e94c446329c251.

Reply:
Awesome, thank you.